README.md

```diff
@@ -1,7 +1,10 @@
-# Markdown Presentation Viewer for Owncloud Web
+# Markdown Presentation Viewer for ownCloud Web
 
-A markdown presentation viewer for the [Owncloud web](https://owncloud.com/features/web-app/) using the [reveal.js](https://revealjs.com/) library.
-It is an extension for the Owncloud web that allows users to create slide presentations directly from markdown files.
+A markdown presentation viewer for [ownCloud web](https://github.com/owncloud/web/) (the webUI of [oCIS](https://github.com/owncloud/ocis/)) using the [reveal.js](https://revealjs.com/) library.
+
+It allows users to:
+- create slide presentations directly from markdown files
```

Comment:
does this really allow to create presentations? my understanding was that the current version of the app only offers previewing presentations that were created in markdown some other software... I think using the term "generate" or "render" or "preview" is more suitable

Reply:
@individual-it @grgprarup @PrajwolAmatya please have a look at the comment above